src/main/java/org/broadinstitute/hellbender/tools/walkers/bqsr/GatherBQSRReports.java

```diff
@@ -0,0 +1,48 @@
+package org.broadinstitute.hellbender.tools.walkers.bqsr;
+
+import htsjdk.samtools.util.IOUtil;
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.ReadProgramGroup;
+import org.broadinstitute.hellbender.utils.recalibration.RecalibrationReport;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Gathers recalibration reports by adding all observations and errors
+ *
+ * Note: This method DOES NOT recalculate the empirical qualities and quantized qualities. You have to recalculate
+ * them after combining. The reason for not calculating it is because this function is intended for combining a
+ * series of recalibration reports, and it only makes sense to calculate the empirical qualities and quantized
+ * qualities after all the recalibration reports have been combined. Having the user recalculate when appropriate,
+ * makes this method faster
```

Comment:
So I copied this text over directly, but I'm not actually sure it's true/relevant.